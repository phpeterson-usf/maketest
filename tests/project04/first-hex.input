-h $(repo_dir)/first.hex $(TESTS_DIR)/first.s
