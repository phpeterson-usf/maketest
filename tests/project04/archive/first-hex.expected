v2.0 raw
E3A0002A
E12FFF1E
